### DISCUSSION TITLE
Zigbee 20 device network completely dead after reboot

### DISCUSSION THREAD

**Discussion by max-rousseau** [Koenkk/zigbee2mqtt#12289] [General]
Hello All - Hoping someone can help out with this pretty catastrophic failure scenario. Let me provide some background / context to help reflect on troubleshooting:

* I'm running HASSIO with version 2022.4.6 of home assistant
* The zigbee2mqtt service is installed as a supervisor add-on. Currently v1.25.0-1.
* The controller hardware I'm using is a sonoff zigbee 3.0 plus usb stick. I have flashed it with the latest version firmware and saw as much in the logs: `2022-04-26 00:00:21: Coordinator firmware version: '{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20220219,"transportrev":2},"type":"zStack3x0"}'`
* I have double-checked that the USB device defined is correct and in fact able to communicate (log item above is good indicator also).
* My network has 20 devices, a mix of Hue lights, Ikea Wall Switches and Aqara sensors. None of them works now.

Example failure log:
`error 2022-04-26 10:13:16: Publish 'set' 'state' to 'Hue Ambiance Light 1' failed: 'Error: Command 0x001788010b5da159/11 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'No network route' (205))'`

In the mqtt topic, all devices have `linkquality:null`
<img width="489" alt="image" src="https://user-images.githubusercontent.com/67350340/165365039-9e2e7269-9148-434a-a1fe-efb4e05296f4.png">

What I've tried so far:

* Made sure firmware was updated
* Rebooted hardware
* Restarted HA, z2m, mqtt - all and any order really
* Added the following to HA config:
```
  birth_message:
    topic: "homeassistant/status"
    payload: "online"
  will_message:
    topic: "homeassistant/status"
    payload: "offline"
```

I'm kind of at my wits end now. Could really use a hand

**Comment by max-rousseau** [comment]
Quick update - about 15 hours later, it seems that the Hue Motions have returned to the cluster (?!). Those are model [9290012607](https://www.zigbee2mqtt.io/devices/9290012607.html#philips-9290012607). Nothing else.

**Comment by kapabel** [comment]
Hello! Did you end up solving your mess? I am experiencing the same.

**Reply by max-rousseau** [reply]
Partly - I'm not 100% sure that this would be reproducible but here is what I did: 

1) Updated the firmware on the zigbee stick
2) Reset one of the lights and manually re-paired it to the stick
3) Other devices started cascading in and re-connecting after #2 (no idea why)
4) Manually re-pair whatever didn't make it
5) Buy a hue dimmer switch to reset / touchlink instead of using stick:
https://www.philips-hue.com/en-us/p/hue-dimmer-switch--latest-model-/046677562779

> Bulbs can also be reset if you have an official Philips Hue Dimmer Switch at home. With the light in question powered on, hold the Dimmer Switch close to it and hold down the On and Off buttons together for 10 seconds or so. When the light blinks and then stops blinking, the bulb is reset.

So far with the new firmware the network has been rock solid for last 3 months.

Good luck.
